internal/server/grpc/grpc.go

```diff
@@ -90,7 +90,7 @@ func (s *Server) Start() error {
 	creds := grpc.Creds(insecure.NewCredentials())
 
 	switch {
-	case s.Config.AttestedTLS:
+	case s.Config.AttestedTLS && s.Config.CertFile != "" && s.Config.KeyFile != "":
```

Comment:
we should make it optional, 
- no tls
- tls
- mtls
- atls
- matls

Reply:
@SammyOina 
default case handles notls
first case handles atls with optional mtls
second case handles atls only
third case handles tls with optional mtls